scenariolib/event_click.go

```diff
@@ -97,13 +97,14 @@ func (ce *ClickEvent) Execute(v *Visit) error {
 	}
 
 	if ce.clickRank == -1 { // if rank == -1 we need to randomize a rank
-		ce.clickRank = 0
 		// Find a random rank within the possible click values accounting for the offset
-		if v.LastResponse.TotalCount > 1 {
-			topL := Min(v.LastQuery.NumberOfResults, v.LastResponse.TotalCount)
-			rndRank := int(math.Abs(rand.NormFloat64()*2)) + ce.offset
-			ce.clickRank = Min(rndRank, topL-1)
-		}
+		ce.clickRank = int(math.Abs(rand.NormFloat64()*2)) + ce.offset
```

Comment:
Not sure I understand the `NormFloat64()*2` here. 
Seems to me that `ce.clickRank = ce.offset + rand.Intn(v.LastResponse.TotalCount - ce.offset)` would do the job here.

Reply:
It's to have a normal distribution between 0 and max, centered around 2 +- the offset.

https://golang.org/pkg/math/rand/#NormFloat64

That method doesn't exist with Integer

Reply:
sure. ok.